impeller/tools/xxd.py

```diff
@@ -60,7 +60,9 @@ def main():
   with open(args.source, 'rb') as source, open(output_source, 'w') as output:
     data_len = 0
     output.write(f'#include "{output_header_basename}"\n')
-    output.write(f'const unsigned char impeller_{args.symbol_name}_data[] =\n')
+    output.write(
+        f'alignas(8) const unsigned char impeller_{args.symbol_name}_data[] =\n'
```

Comment:
This should probably be `alignas(long double)`.

Reply:
Ooh, [found this](https://en.cppreference.com/w/cpp/types/max_align_t). So `alignas(std::max_align_t)` for maximum portability.

Reply:
done